pallet/src/migrations/v0_to_v1.rs

```diff
@@ -0,0 +1,56 @@
+use crate::Config;
+use frame_support::log;
+use frame_support::{
+    traits::{Get, GetStorageVersion, PalletInfoAccess, StorageVersion},
+    weights::Weight,
+};
+use sp_std::prelude::*;
+
+frame_support::generate_storage_alias!(
+    Aleph, OldSessionForValidatorsChange => Value<Option<u32>>
+);
+
+frame_support::generate_storage_alias!(
+    Aleph, OldValidators<T: Config> => Value<Option<Vec<T::AccountId>>>
+);
+
+pub fn migrate<T: Config, P: GetStorageVersion + PalletInfoAccess>() -> Weight {
+    let on_chain_storage_version = <P as GetStorageVersion>::on_chain_storage_version();
+    let new_storage_version = crate::STORAGE_VERSION;
+
+    if on_chain_storage_version == StorageVersion::default() && new_storage_version == 1 {
+        log::info!(target: "pallet_aleph", "Running migration from STORAGE_VERSION 0 to 1");
+
+        let mut writes = 0;
+        let _ = OldSessionForValidatorsChange::translate(|old| match old {
```

Comment:
hopefully, it will always return `Ok(_)`, but maybe it is worth verifying?

Reply:
Good point, added +  logging